src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import SubHeader from '../../components/SubHeader/SubHeader'
+import Carousel from '../../components/Carousel/Carousel'
+import categoryImage1 from '../../assets/Images/skincare-2.jpg'
+import categoryImage2 from '../../assets/Images/tampons.jpg'
+import categoryImage3 from '../../assets/Images/makeup-img-2.jpg'
+import categoryImage4 from '../../assets/Images/hair-care-img.jpg'
```

Comment:
add these images into the public folder and then access as a URL, google this 'accessing public folder files in React' you will understand what I'm talking about.

Reply:
Thanks for the suggestion. Will surely look into this.